### PR TITLE
reactxp-navigator 1.0.10 uwp support

### PR DESCRIPTION
**Pull request by dryganets**

### CODE REVIEW

File: extensions/navigation/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "reactxp-navigation",
-  "version": "1.0.6",
+  "version": "1.0.9",
```
Comment (deregtd) [inline extensions/navigation/package.json]: 7 comes after 6. :)

Comment (dryganets) [reply extensions/navigation/package.json]: It has all versions in.

**Comment by erictraut** [comment]
@dryganets, is this PR still required? Please review my conflict resolution and let me know if it's ready for merging.

**Comment by dryganets** [comment]
@erictraut all conflicts are resolved.

**Pull request merged**